jsapp/js/account/billingContextProvider.component.tsx

```diff
@@ -1,26 +1,22 @@
 import React, {ReactNode} from 'react';
 import {UsageContext, useUsage} from 'js/account/usage/useUsage.hook';
 import {ProductsContext, useProducts} from 'js/account/useProducts.hook';
-import {
-  OrganizationContext,
-  useOrganization,
-} from 'js/account/organizations/useOrganization.hook';
 import sessionStore from 'js/stores/session';
+import {useOrganizationQuery} from 'js/account/stripe.api';
 
 export const BillingContextProvider = (props: {children: ReactNode}) => {
+  const orgQuery = useOrganizationQuery();
+
   if (!sessionStore.isLoggedIn) {
     return <>{props.children}</>;
   }
-  const [organization, reloadOrg, orgStatus] = useOrganization();
-  const usage = useUsage(organization?.id);
+  const usage = useUsage(orgQuery.data?.id || null);
   const products = useProducts();
   return (
-    <OrganizationContext.Provider value={[organization, reloadOrg, orgStatus]}>
       <UsageContext.Provider value={usage}>
```

Comment:
This should be reindented after wrapper node was removed

Reply:
good catch :ok_hand: 

There is several linter changes in every file, so I got carried away hitting "save without formatting" everywhere. Frankly linter didn't bother about this indent. Anyways that's out of scope until we figure out linter for the project as a whole